cdap-watchdog-api/src/main/java/io/cdap/cdap/api/metrics/MetricValue.java

```diff
@@ -24,12 +24,64 @@
   private MetricType type;
   private long value;
 
+  // following fields to support distribution aka histogram/event metrics
+  /*
+  Based on https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TypedValue#Exponential
+   */
+  public static final int GROWTH_FACTOR = 2;
+
+  // Total buckets = NUM_FINITE_BUCKETS +2 which will be 64. A long number
+  // maybe used as a mask to indicate which buckets have non zero bucket counts
```

Comment:
Use javadoc format

Reply:
Thanks. I updated it and probably forgot to amend.